recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "aiidalab" %}
-{% set version = "22.11.0" %}
+{% set version = "23.3.0" %}
 
 package:
   name: {{ name|lower }}
```

Comment:
See `sha256` below, need to be `4f4dd5f5635477ccc4a23c08a7ff03e3fbf206c3dc07f19338c5bedd94cda54a`

Reply:
You can move it to the head so we don't forget in the future, see https://github.com/conda-forge/plumpy-feedstock/blob/main/recipe/meta.yaml

Reply:
Good idea, done.